src/components/LoginPage/Slide5.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import imageMindmap from 'components/LoginPage/images/slide-5-mindmap.svg';
+import arrow from 'components/LoginPage/images/arrow.svg';
+import { Img, Caption } from 'components/LoginPage/slideComponents';
+
+const Slide = styled.section`
+  position: relative;
+  .hint-old-notes {
+    position: absolute;
+    right: 5%;
+    width: 22%;
+    height: 11%;
+    top: 22%;
+    img {
+      width: 12%;
+      position: absolute;
+      bottom: 2%;
+      left: 44%;
+    }
+  }
+  .hint-recent-notes {
+    position: absolute;
+    left: 13%;
+    bottom: 3%;
+    width: 26%;
+    height: 13%;
+    img {
+      width: 12%;
+      position: absolute;
+      top: 2%;
+      left: 44%;
+    }
+  }
+`;
+const H3 = styled.h3`
+  padding-bottom: 0;
+  margin-top: 0;
+  margin-bottom: 0;
+  font-size: 4vmin;
```

Comment:
All other font-sizes look like `calc(16px + (30 - 16)*(100vw - 375px)/(1200 - 375));`, but this one is `4vmin;`. Why? Looks weird :)

Reply:
I found that used `vmin` units is more appropriative to our task. Here some description 
[Specification](https://www.w3.org/TR/css-values-4/#vmin)
[Vmin on WebReference](https://webref.ru/css/unit/vmin)

Done in commit [db8d5d2](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/db8d5d213e208ca30de77f1c6674c40053f31f87)